.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will build a Swift project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-swift
+
+name: Ubuntu
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    container: swift:5.7-focal
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Resolve dependencies
+      run: swift package resolve 
+    - name: Build
+      run: swift build
+    - name: Install ganache
+      run: npm install ganache --global
+    - name: Start ganache in background
+      run: ganache &
+    - name: Wait till ganache starts
+      run: sleep 1
+    - name: install nc
+      run: apt-get update && apt-get install -y netcat
+    - name: Ping
+      run: nc -vz 127.0.0.1 8545
+    - name: Run tests
+      run: swift test
+    # - name: Run local tests
+      # run: swift test --skip-build -c debug --filter localTests
+    # - name: Run remote tests
+      # run: swift test --skip-build -c debug --filter remoteTests
```

Comment:
Since `swift test` is executed on line 38 I think there is no need to keep these comments.

Reply:
I guess it's temporary, commented one is those that now running on macOS12 one.

Reply:
The commented out lines are indeed from the macOS script. I'm wasn't sure if it made sense to split the unit tests in local and remote, or to just run all tests at once. I'll remove lines 39-42